include/api/wasmedge/wasmedge.h

```diff
@@ -51,13 +51,22 @@ typedef struct WasmEdge_String {
   const char *Buf;
 } WasmEdge_String;
 
-/// Opaque struct of WASM execution result.
+/// WasmEdge result struct.
 typedef struct WasmEdge_Result {
-  uint8_t Code;
+  uint8_t Prefix;
+  uint8_t Code1;
+  uint8_t Code2;
+  uint8_t Code3;
 } WasmEdge_Result;
```

Comment:
To avoid the `endianess` issue, it will be better to use `uint32_t` directly and provide some helper functions to get and set the internal values.